### PR TITLE
sot-110-chapter-2

### PR DESCRIPTION
**Pull request by Tranduy1dol**
# Check List

- [x] Did you check your grammar / reader proof your document using ChatGPT?
- [x] Don't forget to squash commits into meaningful chunks before merging
- [x] After merging this, please update your task status!
- [x] Did you verify that your math formulas are rendered correctly?

### CODE REVIEW

File: articles/zcash-protocol/chapter_2.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+---
+
+- $\mathbb{B}$ : the type of bit values.
+- $\mathbb{B}^{\mathbb{Y}}$ : the type of byte values
+- $\mathbb{N}$ : the type of nonnegative integers.
+- $\mathbb{N}^{+}$ : the type of positive integers.
+- $\mathbb{Z}$ : the type of integers.
+- $\mathbb{Q}$ : the type of rational.
+- $x\ : \ T$ : $x$ has type $T$.
```
Comment (bjergsen243) [inline articles/zcash-protocol/chapter_2.md:16]: $x\ \colon \ T$ means $x$ has type $T$.

Comment (bjergsen243) [reply articles/zcash-protocol/chapter_2.md:16]: use "\colon" instead of ":"

Comment (bjergsen243) [reply articles/zcash-protocol/chapter_2.md:16]: Create a table with two columns. First column is Denoted, second column is Meaning

File: articles/zcash-protocol/chapter_2.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+| $a$ mod $q$, for $a:\mathbb{N}$ and $q:\mathbb{N}^{+}$                                           | The remainder on dividing $a$ by $q$.                                                                                                                                                                              |
+| $a\ \oplus\ b$                                                                                   | The bitwise-exclusive-or of $a$ and $b$. These are defined on integers, elementwise on equal-length sequences of integers, according to context.                                                                   |
+| $a\ \&\ b$                                                                                       | The bitwise-and of $a$ and $b$ . These are defined on integers, elementwise on equal-length sequences of integers, according to context.                                                                           |
```
Comment (bjergsen243) [inline articles/zcash-protocol/chapter_2.md:62]: $a\ \\& \ b$  
 "\\&"

File: articles/zcash-protocol/chapter_2.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+| $x \ \xleftarrow{R}\ f(s)$                                                                       | With $f:S \xrightarrow{R} T$ and $s: S$, sampling a variable $x:T$ from the output of $f$ applied to $s$.                                                                                                          |
+| $x:X$, $y:Y$, $f:X\times Y\to Z$                                                                 | Initial arguments to a function or randomized algorithm.                                                                                                                                                           |
+| $f(x,y)$                                                                                         | Can be written $f_x(y)$.                                                                                                                                                                                           |
```
Comment (bjergsen243) [inline articles/zcash-protocol/chapter_2.md:23]: Can be written as

**Review by bjergsen243 — APPROVED** [review]

**Review by VanhGer — APPROVED** [review]

**Review by hduoc2003 — APPROVED** [review]

**Pull request merged**